internal/sourcemap/sourcemap.go

```diff
@@ -209,7 +209,6 @@ func (offset *LineColumnOffset) AdvanceBytes(bytes []byte) {
 		case '\r', '\n', '\u2028', '\u2029':
 			// Handle Windows-specific "\r\n" newlines
 			if c == '\r' && len(bytes) > 0 && bytes[0] == '\n' {
```

Comment:
I'm not 100% sure I follow the logic of this, is special behaviour needed for Windows because the generated code always uses `\n`? Otherwise, couldn't it be mapped 1:1?

Reply:
Yes, I'm pretty sure the previous logic was only correct if the whole file was printed with `\r\n`, but as you correctly point out, we only print `\n`.